/**:
    ros__parameters:
        num_max_iterations: 3
        filter_size_surf_min: 0.5
        filter_size_map_min: 0.0             # 暂时未用到，代码中为0， 即倾向于将降采样后的scan中的所有点加入map
        esti_plane_threshold: 0.1        # default=0.1

        ivox:
            grid_resolution: 0.5        # default=0.2
            nearby_type: 18             # 6, 18, 26
        
        common:
            lid_topic:  "/livox/lidar"
            imu_topic:  "/livox/imu"
            time_sync_en: false         # ONLY turn on when external time synchronization is really not possible

        preprocess:
            lidar_type: 1                # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR, 4 for any other pointcloud input
            num_scans:  4
            blind: 0.1
            time_scale: 1e-3
            feature_extract_enable: false
            point_filter_num: 3

        mapping:
            acc_cov: 0.1
            gyr_cov: 0.1
            b_acc_cov: 0.0001
            b_gyr_cov: 0.0001
            extrinsic_est_en:  true      # true: enable the online estimation of IMU-LiDAR extrinsic
            extrinsic_t: [ -0.011, -0.02329, 0.04412 ]
            extrinsic_r: [ 1., 0., 0.,
                            0., 1., 0.,
                            0., 0., 1. ]

        publish:
            path_publish_en: false
            scan_publish_en: true       # false: close all the point cloud output
            scan_effect_pub_en: true    # true: publish the pointscloud of effect point
            dense_publish_en: false       # false: low down the points number in a global-frame point clouds scan.
            scan_body_pub_en: true  # true: output the point cloud scans in IMU-body-frame

        pcd:
            save_pcd_en: true
            save_interval: -1                 # how many LiDAR frames saved in each pcd file; 
                                        # -1 : all frames will be saved in ONE pcd file, may lead to memory crash when having too much frames.
